cypress/cypress/support/project-commands/project.ts

```diff
@@ -68,7 +66,7 @@ const deleteProjectsWithPrefix = (projectNamePrefix: string, maxNumberOfDeletes
         if (sessionStorageKey.startsWith('oidc.user')) {
             // Use OIDC header to connect to the project API
             cy.request({
-                url: `${BASE_URL}/project/`,
+                url: LAMBDA_ENDPOINT ? `${LAMBDA_ENDPOINT}/project/` : `${BASE_URL}/project/`,
```

Comment:
Is this the only place we need this change? We probably need a comment on that constant explaining when you should use `LAMBDA_ENDPOINT` vs `BASE_URL`